README.md

```diff
@@ -93,6 +93,34 @@ others, it does, specifically:
 If you'd like to see support for these commands as well, feel free to submit a
 pull request!
 
+## Load Balancing (advanced)
+
+By default, `lerna-parallelism` gives each partition an approximately equal number
+of packages by chunking an alphabetized array. Depending on the number of tests 
+each of your packages has, you may find this results in imbalanced runtimes between 
+partitions.
+
+To address this, `lerna-parallelism` offers a load balancing mode which uses a
+weighted round-robin algorithm, informed by a weight "hint" specified in each 
+package's `package.json`.
+
+This adds two additional CLI options in addition to those above:
+
+- `--loadBalance`: Toggles load-balancing mode. Defaults to
+  `false`.
+- `--packageTestWeightKey myProjectWeight`: The lookup key used to read the project's
+  weight from its package.json. Defaults to `lernaPackageTestWeight`.
```

Comment:
Is there a reason you chose `--packageTestWeightKey` over just `--packageWeightKey`?

Would you be okay with changing this?

```suggestion
- `--packageWeightKey myProjectWeight`: The lookup key used to read the project's
  weight from its `package.json`. Defaults to `lernaPackageWeight`.
```